core/llm/index.ts

```diff
@@ -414,9 +415,11 @@ export abstract class BaseLLM implements ILLM {
             e.code === "ECONNREFUSED" &&
             e.message.includes("http://127.0.0.1:11434")
           ) {
-            throw new Error(
-              "Failed to connect to local Ollama instance, please ensure Ollama is both installed and running. You can download Ollama from https://ollama.ai.",
-            );
+            const message = (await isOllamaInstalled()) ?
+              "Failed to connect to local Ollama instance. Ollama appears to be stopped. It needs to be running." :
```

Comment:
Three sentences feels a bit hard to read, and reads a bit abruptly. Reducing it down to two sentences would be easier to digest.

```suggestion
              "Unable to connect to local Ollama instance. Ollama may not be running." :
```